.circleci/config.yml

```diff
@@ -1,6 +1,17 @@
 references:
+  docker-init: &docker-init
+    entrypoint:
+      - docker-init
```

Comment:
It means like `--init` option of Docker. Handle signal to Chrome executed by Puppeteer correctly, and prevent to remain zombie processes while running CI.